shared/src/main/java/org/geysermc/hydraulic/item/ItemPackModule.java

```diff
@@ -131,13 +132,10 @@ private void onDefineCustomItems(PackEventContext<GeyserDefineCustomItemsEvent,
         DefaultedRegistry<Item> registry = BuiltInRegistries.ITEM;
         for (Item item : items) {
             ResourceLocation itemLocation = registry.getKey(item);
-            DataComponentMap components = item.components();
 
-            // Check if the item has a 2D icon
-            //boolean flatIcon = itemsWith2dIcon.contains(itemLocation.toString());
             NonVanillaCustomItemDefinition.Builder customItemDefinition = NonVanillaCustomItemDefinition.builder(
                     Identifier.of(itemLocation.toString()),
-                    Identifier.of(itemLocation.toString()/* + (flatIcon ? "_item" : "")*/), // Lets do a little test... Doesn't seem needed
+                    Identifier.of(itemLocation.toString()),
```

Comment:
Have you tested with a modded flower or similar that has a 2d item texture but 3d model?